src/docgen/view/property.ts

```diff
@@ -38,6 +38,8 @@ export class Property {
       md.lines('');
     }
 
+    md.code(this.transpile.language.toString(), this.transpiled.signatureOrGetter);
```

Comment:
Since constants currently delegate to properties, this applies to constants as well:

https://gist.github.com/Chriscbr/4de488d36ece0d124149ffdb70688230#cfn_resource_type_name-

It shouldn't

Reply:
Good callout.